source/guides/stacks/dars.rst

```diff
@@ -172,6 +189,487 @@ DARS Usecase example
+      --master k8s://${MASTER_ADDRESS} \
+      --deploy-mode cluster \
+      --name spark-pi \
+      --class org.apache.spark.examples.SparkPi \
+      --conf spark.executor.instances=2 \
+      --conf spark.kubernetes.container.image=${DARS_K8S_IMAGE} \
+      --conf spark.kubernetes.driver.pod.name=spark-pi-driver \
+      --conf spark.kubernetes.namespace=default \
+      --conf spark.kubernetes.authenticate.driver.serviceAccountName=spark-serviceaccount \
+      local:///usr/share/apache-spark/examples/jars/spark-examples_2.12-2.4.0.jar
+
+
+#. Check the Job. Read the logs and look for the Pi result:
+
+   .. code-block:: bash
+
+      kubectl logs spark-pi-driver | grep "Pi is roughly"
+
+   You should see something like:
+
+   .. code-block:: console
+
+      Pi is roughly 3.1418957094785473
+
+More information about spark-submit configuration is available in the  `running-on-kubernetes`_ documentation.
+
+
+Troubleshooting
+***************
+
+Dropped or refused connection
+=============================
+
+If Pyspark / Spark-shell warns of a dropped connection exception or Connection refused, check if the `HADOOP_CONF_DIR` environment variable is set. These APIs assume they will use Hadoop Distributed File System.
+You can unset `HADOOP_CONF_DIR` and use Spark RDDs, or start Hadoop services and then create your directories and files as required using hdfs.
+
+It is also possible to change the file system to local without unsetting `HADOOP_CONF_DIR` as is further described here:
+
+.. code-block:: bash
+
+   pyspark --conf "spark.hadoop.fs.defaultFS=file:///"
+
+.. code-block:: bash
+
+   spark-shell --conf "spark.hadoop.fs.defaultFS=file:///"
+
+Using Spark with proxy settings
+===============================
+
+There are two ways to work with proxies:
+
+#. Add the following line to  :file:`$SPARK_CONF_DIR/spark-defaults.conf` for both `spark.executor.extraJavaOptions` and `spark.driver.extraJavaOptions` variables:
+
+.. code-block:: console
+
+   -Dhttp.proxyHost=<URL> -Dhttp.proxyPort=<PORT> -Dhttps.proxyHost=<URL> -Dhttps.proxyPort=<PORT>
+
+
+
+#. Give the proxies URL and Port as a configuration parameter
+
+.. code-block:: bash
+
+   pyspark --conf "spark.hadoop.fs.defaultFS=file:///" --conf "spark.driver.extraJavaOptions=-Dhttp.proxyHost=example.proxy -Dhttp.proxyPort=111 -Dhttps.proxyHost=example.proxy -Dhttps.proxyPort=112"
+
+.. code-block:: bash
+
+   spark-shell --conf "spark.hadoop.fs.defaultFS=file:///" --conf "spark.driver.extraJavaOptions=-Dhttp.proxyHost=example.proxy -Dhttp.proxyPort=111 -Dhttps.proxyHost=example.proxy -Dhttps.proxyPort=112"
+
+
+Known issues
+============
+
+#. There is an exception message `Unrecognized Hadoop major version number: 3.2.0 at org.apache.hadoop.hive.shims.ShimLoader.getMajorVersion.`
+
+This exception can be disregarded because DARS does not use hadoop.hive.shims. Hive binaries installed from Apache on |CL| with JDK11 does not work, this is an issue reported on Hive's Jira.
```

Comment:
```suggestion
This exception can be disregarded because DARS does not use hadoop.hive.shims. Hive binaries installed from Apache on |CL| with JDK11 does not work at the time of this writing.
```